README.rst

```diff
@@ -951,16 +951,21 @@ among the most popular alternatives:
   <https://pypi.org/project/devpi/>`_.
   (version: 2.1.4, access date: 8/3/2015)
 
+- Check this SO question: `How to roll my own pypi
+  <http://stackoverflow.com/questions/1235331/how-to-roll-my-own-pypi>`_
```

Comment:
Why do you think this is necessary? It is a little vague why would it be helpful to read this question. I would say either elaborate on why or remove.